test/services/pilot/create-asset-client/client.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"fybrik.io/fybrik/pkg/model/datacatalog"
+)
+
+func main() {
+	// the below logic is inspired from https://levelup.gitconnected.com/consuming-a-rest-api-using-golang-b323602ba9d8
+	request := datacatalog.CreateAssetRequest{
+		DestinationCatalogID: "test",
+	}
+
+	postBody, _ := json.Marshal(request)
+	responseBody := bytes.NewBuffer(postBody)
+
+	fmt.Println("Making a HTTP Request")
+	client := &http.Client{}
+	req, err := http.NewRequest("POST", "http://katalog-connector:80/createAssetInfo", responseBody)
+	if err != nil {
+		fmt.Print(err.Error())
+	}
+	req.Header.Add("Accept", "application/json")
+	req.Header.Add("Content-Type", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		fmt.Print(err.Error())
+	}
+	defer resp.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
I believe io.ReadAll should be used according to lint (@Raviv-S ?)

Reply:
[Yes, ioutil.ReadAll is deprecated](https://pkg.go.dev/io/ioutil#ReadAll`)

Reply:
Fixed. Thanks for pointing out. Now using io.readAll instead of ioutil.readAll. Please check.